src/omv/modules/py_helper.h

```diff
@@ -15,14 +15,14 @@ extern const mp_obj_fun_builtin_var_t py_func_unavailable_obj;
 image_t *py_helper_arg_to_image_mutable(const mp_obj_t arg);
 image_t *py_helper_arg_to_image_not_compressed(const mp_obj_t arg);
 image_t *py_helper_arg_to_image_grayscale(const mp_obj_t arg);
-image_t *py_helper_keyword_to_image_mutable(uint n_args, const mp_obj_t *args, uint arg_index,
-                                            mp_map_t *kw_args, mp_obj_t kw, image_t *default_val);
-image_t *py_helper_keyword_to_image_mutable_mask(uint n_args, const mp_obj_t *args, uint arg_index,
-                                                 mp_map_t *kw_args);
-image_t *py_helper_keyword_to_image_mutable_color_palette(uint n_args, const mp_obj_t *args, uint arg_index,
-                                                          mp_map_t *kw_args);
-image_t *py_helper_keyword_to_image_mutable_alpha_palette(uint n_args, const mp_obj_t *args, uint arg_index,
-                                                          mp_map_t *kw_args);
+image_t *py_helper_keyword_to_image(uint n_args, const mp_obj_t *args, uint arg_index,
+                                    mp_map_t *kw_args, mp_obj_t kw, image_t *default_val);
+image_t *py_helper_keyword_to_image_mask(uint n_args, const mp_obj_t *args, uint arg_index,
+                                         mp_map_t *kw_args);
+image_t *py_helper_keyword_to_image_color_palette(uint n_args, const mp_obj_t *args, uint arg_index,
+                                                  mp_map_t *kw_args);
+image_t *py_helper_keyword_to_image_alpha_palette(uint n_args, const mp_obj_t *args, uint arg_index,
+                                                  mp_map_t *kw_args);
```

Comment:
```suggestion
```
Can you please remove those functions and just call `py_helper_keyword_to_image` directly ? They just seem useless, and when it's time to switch to mp_parse_args, it will be easier to just fix one function (which will accept a single object).

Reply:
Done